vue-social/src/views/Login.vue

```diff
@@ -48,6 +48,12 @@
                 (user) => {
                   this.username = user.user.email;
                   localStorage.setItem("username", this.username);
+                  firebase.auth().currentUser.getIdToken(true).then(function (idToken) {
```

Comment:
use => syntax instead of function(idtoken)

Reply:
done